demos/fab.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
it may be good to have one example in the demo of a FAB positioned in the bottom-left corner, to show how easy it is to customize the behavior that way.